cmd/mesh/manifest-diff.go

```diff
@@ -33,11 +33,23 @@ const YAMLSuffix = ".yaml"
 type manifestDiffArgs struct {
 	// compareDir indicates comparison between directory.
 	compareDir bool
+	// selection specifies the resources to compare, it's comma-separated of resource indicators
+	// in the form of "<kind>:<namespace>:<name>,<kind>:<namespace>:<name>".
+	selection string
+	// ignore specifies the resources to ignore, it's comma-separated of resource indicators
+	// in the form of "<kind>:<namespace>:<name>,<kind>:<namespace>:<name>".
```

Comment:
ignoreResources ignores all listed items during comparison. It uses the same list format as selectResources.

Reply:
done.